src/index.md

```diff
@@ -1547,7 +1529,65 @@ The Skara tooling includes support for backports. [The official Skara documentat
 [To the top](#){.boxheader}
 :::
 
-# HotSpot development
+# Release Notes
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [release-note label description](#release-note)
+:::
+
+Release notes for a product (e.g. the JDK) are part of the release deliverables. They describe changes that are important for a user of the product to know about. This is usually things that may affect the user's decision to upgrade to the specific version.
```

Comment:
"to know about.  This is usually" -> "to know.  These are usually"

Reply:
Fixed.